graphx/pom.xml

```diff
@@ -50,6 +50,12 @@
       <artifactId>scalacheck_${scala.binary.version}</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+        <version>4.11</version>
```

Comment:
I don't think this version should be specified here; it should be inherited from the parent POM or set via a configuration or propery.